kratos/modified_shape_functions/triangle_2d_3_ausas_incised_shape_functions.cpp

```diff
@@ -0,0 +1,152 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "modified_shape_functions/triangle_2d_3_ausas_incised_shape_functions.h"
+
+namespace Kratos
+{
+
+/// Triangle2D3AusasIncisedShapeFunctions implementation
+/// Default constructor
+Triangle2D3AusasIncisedShapeFunctions::Triangle2D3AusasIncisedShapeFunctions(const GeometryPointerType pInputGeometry,
+        const Vector& rNodalDistancesWithExtrapolated, const Vector& rExtrapolatedEdgeRatios) :
+    Triangle2D3AusasModifiedShapeFunctions(pInputGeometry, rNodalDistancesWithExtrapolated), mExtraEdgeRatios(rExtrapolatedEdgeRatios) {};
+
+/// Destructor
+Triangle2D3AusasIncisedShapeFunctions::~Triangle2D3AusasIncisedShapeFunctions() {};
+
+/// Turn back information as a string.
+std::string Triangle2D3AusasIncisedShapeFunctions::Info() const {
+    return "Triangle2D3N Ausas incised shape functions computation class.";
+}
```

Comment:
```suggestion
std::string Triangle2D3AusasIncisedShapeFunctions::Info() const
{
    return "Triangle2D3N Ausas incised shape functions computation class.";
}
```
Formatting (same in the other methods).